<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_freq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>频率：0.20GHz</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_freq">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>18</y>
     <width>1131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimum">
    <number>20</number>
   </property>
   <property name="maximum">
    <number>1800</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_freq_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>135</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>↑ 2GHz：链路切换</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_freq_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>194</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>↑ 6GHz：链路切换/LH切换</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_ctrl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_tx">
    <attribute name="title">
     <string>Tx控制</string>
    </attribute>
    <layout class="QVBoxLayout" name="tab_tx_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_ctrlTx">
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="groupBox_IDAC">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>IDAC</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>31</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>P</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>31</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>N</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_T_IDAC_IP">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>66</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_T_IDAC_IN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>66</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_T_IDAC_QP">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>103</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_T_IDAC_QN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>103</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>103</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Q</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>66</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>121</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>衰减</string>
        </property>
        <widget class="QCheckBox" name="checkBox_T_att2db">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>30</y>
           <width>62</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>2dB</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_T_att10db">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>70</y>
           <width>72</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>10dB</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_gain">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>221</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>增益</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>32</y>
           <width>64</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>基带增益</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_T_BBGain">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>-6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12dB</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>64</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>滤波增益</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_T_filterGain">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>0dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12dB</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_rx">
    <attribute name="title">
     <string>Rx控制</string>
    </attribute>
    <layout class="QVBoxLayout" name="tab_rx_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_ctrlRx">
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>160</y>
          <width>81</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>衰减</string>
        </property>
        <widget class="QCheckBox" name="checkBox_R_att2db">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>30</y>
           <width>62</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>2dB</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_R_att10db">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>70</y>
           <width>72</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>10dB</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_IDAC_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>IDAC LB-&gt;1  HB-&gt;TIA</string>
        </property>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>31</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>P</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>31</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>N</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC1_IP">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>66</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC1_IN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>66</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC1_QP">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>103</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC1_QN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>103</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_21">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>103</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Q</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>66</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_gain_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>361</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>增益</string>
        </property>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>32</y>
           <width>64</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>基带增益</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_R_BBGain">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>30</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>-6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12dB</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_24">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>一级滤波增益</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_R_filterGain1">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>0dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12dB</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_25">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>二级滤波增益</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_R_filterGain2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>70</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>0dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12dB</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_30">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>70</y>
           <width>64</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>TIA增益</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_R_TIAGain">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>70</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>30dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24dB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18dB</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_IDAC_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>IDAC LB-&gt;2  HB-&gt;FIL</string>
        </property>
        <widget class="QLabel" name="label_26">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>31</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>P</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_27">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>31</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>N</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC2_IP">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>66</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC2_IN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>66</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC2_QP">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>103</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_R_IDAC2_QN">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>103</y>
           <width>75</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_28">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>103</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Q</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_29">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>66</y>
           <width>16</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_BBBW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>251</width>
     <height>66</height>
    </rect>
   </property>
   <property name="title">
    <string>带宽</string>
   </property>
   <widget class="QRadioButton" name="radioButton_BBBW100M">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>76</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>100M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_BBBW250M">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>31</y>
      <width>76</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>250M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_BBBW500M">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>31</y>
      <width>76</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>500M</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget_console">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_txConfigSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TX配置发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rxConfigSend">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RX配置发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_consoleClear">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>510</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>731</width>
     <height>671</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_comm">
    <attribute name="title">
     <string>通信</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_runStop">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>启动/停止</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_radar">
    <attribute name="title">
     <string>雷达</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_adversarial">
    <attribute name="title">
     <string>对抗</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
